christfellowshipchurch/apollos-api
# Update Christmas Feature

### About
This PR increases number of items for 25 Days of Hope `ActionListFeature`.

### Test Instructions
* Run api and web locally
* Go to `/25days`
* You'll see now that all the items from the 25 Days of Hope category are showing.

### Screenshot
![image](https://user-images.githubusercontent.com/46049974/142873754-5b8d0840-b7df-46d7-a20a-a708c758a0a2.png)

### Closes Tickets
[CFDP-1826]


[CFDP-1826]: https://christfellowshipchurch.atlassian.net/browse/CFDP-1826?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
